### ISSUE TITLE
unnecessary .xml suite file created under /tmp when launching xml suite

### ISSUE THREAD

**Issue by missedone** [cbeust/testng-eclipse#259]
see more details: http://stackoverflow.com/questions/37560609/where-to-put-testng-xml-file-in-ant-java-web-project

**Comment by sbmc7** [comment]
Sample java Project to test TestNG Framework

**Comment by missedone** [comment]
@sbmc7

I can't re-produce it with your testng project: https://github.com/sbmc7/testng.git

here is my output: 
```
[TestNG] Running:
  /Users/nick/workspace/myproject/test-ng/testng.xml

Going to execute test Cases

===============================================
Suite1
Total tests run: 1, Failures: 0, Skips: 0
===============================================
```

we can see it running with the testng xml suite file under the project, rather than creating a new one at '/tmp' .

**Comment by sbmc7** [comment]
@missedone 
I think, I got the issue. Before running the test case by xml suite I executed it by Java class TestCaluculator. So the testNG configuration set it to test.TestCaluculator class and every time I execute .xml by right click it actually execute old class configuration instead of xml suite.  I changed the TestNG configuration from **class** to **suite**  and it worked.

Thank you very much for your kind support to solve the issue.
